src/core/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,136 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React, { ReactNode } from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import Color from './Color';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/color-picker.css';
+
+export const DefaultColors = [
```

Comment:
oh, I meant a little bit simpler constant here :D
You could just have simple list of color info object, not give the `Color` component itself. People might want to reuse this list for some different purposes...
`export const DefaultColors = [{color='#FFFFFF' name='WHITE'}, {color='#5A6973' name='GREY'}, ...]`
Then need to think if we want to export this through barrel file in here or maybe move this constant to common...

Reply:
I fixed it.
I'm not sure if it is used anywhere else so I just left it in color picker for now.